README.md

```diff
@@ -185,41 +311,65 @@ $billingAddress->setPhone('0987654321');
-// the ID could also come from the backend.
-function getSessionId() {
-    var sessionId = Cookies.get('sessionId');
-    if(!sessionId) {
-        sessionId = '';
-        var possible = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
-        
-        for (var i = 0; i < 32; i++) {
-            sessionId += possible.charAt(Math.floor(Math.random() * possible.length));
-        }
-        Cookies.set('sessionId', sessionId);
-    }
-    return sessionId;
-}
+$request = new Request([
+    'session' => [
+        'id' => 'ABC123',
+        'ip' => '127.0.0.1',
+    ],
+    'purchase' => [
+        'id' => 1,
+        'createdAt' => new \DateTime('2017-09-02 12:12:12'),
+        // ...
+    ],
+    // "products" is an array of products
+    'products' => [
+        [
+            'sku' => 'SKU1',
+            'url' => 'http://www.example.com/product-1',
+            'price' => 6025,
+            'category' => 'Shoes',
+            // ...
+        ],
+        [
+            'sku' => 'SKU99',
+            'url' => 'http://www.example.com/product-99',
+            'price' => 2050,
+            'category' => 'Hats',
+            // ...
+        ],
+    ],
+    'payment' => [
+        'bin' => '457173',
+        'last4' => '9000',
+        'avs' => 'Y',
+        'cvv' => 'M',
+        // ...
+    ],
+    // ...
+]);
+```
 
-// Must be defined before adding the trackingCodes
-var trackingCodes = [];
+### Responses
 
-// Inject one or more service's tracking code
-<?= $fraudService->trackingCode(ServiceInterface::PAGE_ALL); ?>
+The [`Response` interface](https://github.com/lxrco/omnifraud-common/blob/master/src/Contracts/ResponseInterface.php) exposes a few concise methods.
 
-// Call the functions with the code
-for(var i in trackingCodes) {
-    if(!trackingCodes.hasOwnProperty(i)) continue;
-    trackingCodes[i](getSessionId());
-}
+* `public function getRequestUid(): string;` - the UID generated by the service for this request. Useful for future updates (see async responses below), cancellations, and generating URLs to the web-view of this fraud request
+* `public function isPending(): bool;` - whether or not this response contains a ready response. Some services will require extra time to manually review a request before giving a score + guarantee.
+* `public function getScore(): ?float;` - if not pending, will contain a score from 0 to 100 (0 = worst, 100 = best)
+* `public function isGuaranteed(): bool;` - if supported by the service, determines whether or not the liability for this order can be shifted to the service. Typically called "Chargeback guarantee".
+
+#### Asynchronous Responses
+
+When you get a response that's pending (`isPending() === true` above), you're not yet capable of retrieving it's score and guarantee. In this case you should make note of the request UID, dispatch a background job, and try updating it again at a later date.
```

Comment:
Dispatching a background job is not a requirement, we can give it as an example, but there are other options to.